### PR TITLE
WW-481 after cnw rebranding test fixes

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'org.openqa.selenium.support.ui.Select'.](https://github.com/Wikia/selenium-tests/blob/0b28a00f10e0163d5ab828e15e7e0c8b463a428f/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep1.java#L14) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep1.java [hunk 1]
```diff
@@ -57,15 +56,20 @@ public String getWikiName() {
...
+
+    List<WebElement> langList = wikiLanguageList.findElements(By.cssSelector("li:not(.spacer)"));
+
     for (int i = 0; i < langList.size(); i++) {
-      String langDropElement = langList.get(i).getText();
-      if (langDropElement.contains(lang + ":")) {
-        language.selectByIndex(i);
+      WebElement selectedLanguage = langList.get(i);
+      String selectedLanguageText = selectedLanguage.getText();
+      if (selectedLanguageText.contains(lang + ":")) {
```
Comment (rybmat) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep1.java]: can we move this string concatenation outside of loop to not concatenate it with every iteration?

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep2.java [hunk 2]
```diff
@@ -36,11 +37,17 @@ public void describeYourTopic(String description) {
         .log("describeYourTopic", "describe your topic populated with: " + description, true);
   }
 
-  public void selectCategory(String category) {
-    wait.forElementVisible(wikiCategory);
-    Select dropList = new Select(wikiCategory);
-    dropList.selectByVisibleText(category);
-    PageObjectLogging.log("selectCategory", "selected " + category + " category", true, driver);
+  public void selectCategory(int categoryId) {
+    wait.forElementVisible(wikiCategoryDropdown);
```
Comment (rybmat) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep2.java]: clickable

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep1.java [hunk 3]
```diff
@@ -57,15 +56,21 @@ public String getWikiName() {
 
 
   public void selectLanguage(String lang) {
-    scrollAndClick(languageSelectorTrigger);
-    wait.forElementVisible(languageSelector);
-    Select language = new Select(languageSelector);
-    List<WebElement> langList = language.getOptions();
+    wait.forElementVisible(wikiLanguageDropdown);
```
Comment (rybmat) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep1.java]: clickable

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'org.openqa.selenium.support.ui.Select'.](https://github.com/Wikia/selenium-tests/blob/cbb93305b2abeb6f3108b3dc46ec72de388b0c40/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep1.java#L14) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Pull request merged**